docker-compose.services.yml

```diff
@@ -129,8 +137,8 @@ services:
       - APISIX_SESSION_SECRET_KEY=${APISIX_SESSION_SECRET_KEY:-something_at_least_16_characters}
       - APISIX_LOGOUT_URL=${APISIX_LOGOUT_URL:-http://open.odl.local:8065/}
       - MITOL_API_DOMAIN=${MITOL_API_DOMAIN:-open.odl.local}
-      - MITX_ONLINE_UPSTREAM=${MITX_ONLINE_UPSTREAM:-mitxonline.odl.local:8013}
-      - MITX_ONLINE_DOMAIN=${MITX_ONLINE_DOMAIN:-mitxonline.odl.local}
+      - MITX_ONLINE_UPSTREAM=${MITX_ONLINE_UPSTREAM:-host.docker.internal:8013}
+      - MITX_ONLINE_DOMAIN=${MITX_ONLINE_DOMAIN:-host.docker.internal}
```

Comment:
I had to set `MITX_ONLINE_DOMAIN` to `mitxonline.odl.local`. This allows you to access http://mitxonline.odl.local:8065/ in a browser and then, using your already logged in session in APISIX, you are automatically authenticated to use Django admin in MITx Online.

Reply:
I'm not entirely sure what the issue was here, but I've changed so that `docker.host.internal` is not longer used and we just use `mitxonline.odl.local` everywhere.